lib/extensions/ar_migration.rb

```diff
@@ -10,19 +10,24 @@ def self.discover_schema_migrations_ran_class
   end
 
   def self.update_local_migrations_ran(version, direction)
-    return unless schema_migrations_ran_class = discover_schema_migrations_ran_class
-
-    new_migrations = ActiveRecord::SchemaMigration.normalized_versions
-    new_migrations << version if direction == :up
-
-    to_add = (new_migrations - schema_migrations_ran_class.pluck(:version))
-    puts "Seeding :schema_migrations_ran table..." if to_add.size > 1
+    return unless schema_migrations_ran_class = discover_schema_migrations_ran_class(version)
+
+    if direction == :up
+      if version == "20171031010000"
+        new_migrations = ActiveRecord::SchemaMigration.normalized_versions
+        new_migrations << version
+        to_add = (new_migrations - schema_migrations_ran_class.pluck(:version))
```

Comment:
There can't be anything in the `schema_migrations_ran_class` table at this point, so I don't think this is even needed.